### ISSUE TITLE
on-nullable field 'order' to mapping without a default

### ISSUE THREAD

**Issue by ACPK** [bfirsh/django-ordered-model#140]

**Comment by bfirsh** [comment]
Could you explain in more detail what this is about?

**Comment by ivanff** [comment]
self.swap(self.get_ordering_queryset()
                      .filter(**{self.order_field_name + '__lt': getattr(self, self.order_field_name)})
                      .order_by('-' + self.order_field_name))

if all order fields is 0, qs is blank, and all methods up and down not work

**Comment by shuckc** [comment]
Yes it's a known issue that your models need to be ordered from the start. If you retrospectively add ordering using `makemigrations`/`migrate` to existing instances, they all start at zero. There is an open pull request to add a management command to `manage.py` to resolve this.

**Comment by alstr** [comment]
Is this any closer to being resolved?
